Duplicati.Library.Backend.Storx/Access.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+using Duplicati.Library.Utility;
+
+namespace Duplicati.Library.Backend.Storx
+{
+    public class Access
+    {
+        private const string root = "https://storx.io/api/";
+        private const string key = "eyB6K9tfCOxxas7hkFDcZ4ulcgEsJ80V";
+        private const string vi = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+        private const string salttext = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
```

Comment:
It does not seem secure that these values are fixed and stored in the client. As they are publicly known, any encryption performed with them adds no protection (anyone interception the traffic can decrypt). 

Is there an API explanation or documentation somewhere?